src/renderer/App.d.ts

```diff
@@ -47,6 +47,26 @@ type Graph = {
   links: Link[];
 };
 
+export type Service = {
+  build?: string;
+  command?: string;
+  environment?: ReadOnlyObj;
+  image?: string;
+  ports?: string[];
+  volumes?: string[];
+  working_dir?: string;
+  security_opt?: string[];
+  stop_grace_Period?: string;
+  stop_signal?: string;
+  tmpfs?: string[];
+  restart: string;
+  pid: string;
+};
+
+export type ServiceInfo = {
```

Comment:
@dannyscheiner I think this type name should match what it's typing in state.
```
export type Services = {
  [service: string]: Service;
};
```
Then on line 6: 
`services: Services;`